dev-lang/cobol/gnucobol-3.0~rc1.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A free COBOL compiler"
+DESCRIPTION="GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler."
+HOMEPAGE="https://sourceforge.net/projects/open-cobol/"
+COPYRIGHT="2001-2019 Free Software Foundation, Inc"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://downloads.sf.net/open-cobol/gnu-cobol/gnucobol-3.0-rc1.tar.gz"
+CHECKSUM_SHA256="e55aeea6b1f77e763b4cd4b520c78eb06da7671b4dcc76463fd0237dbf2e4816"
+SOURCE_DIR="gnucobol-3.0-rc1"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	gnucobol = $portVersion
+	cmd:cob_config
+	cmd:cobc
+	cmd:cobcrun
+	lib:libcob
```

Comment:
soname is 4.0.0 so it has to be added here (or use libVersionCompat)
It also need a devel:libcob to install the static lib (strange that haikuporter -S doesn't complain about it)